bundles/io/org.eclipse.smarthome.io.transport.mdns/src/main/java/org/eclipse/smarthome/io/transport/mdns/discovery/MDNSDiscoveryService.java

```diff
@@ -62,12 +62,6 @@ public void unsetMDNSClient(MDNSClient mdnsClient) {
 
     @Override
     protected void startBackgroundDiscovery() {
-        scheduler.schedule(new Runnable() {
```

Comment:
The initial scan imho makes a lot of sense as you otherwise simply miss all the devices in the network that are already there. It is nothing expensive and it will bring the best result. "background discovery" simply means that the service does its best to find devices by itself, without waiting for the user to trigger a scan.

> MdnsClient.list() would not initiate a scan

That's why the code executes a `startScan();`.